### PR TITLE
Add documentation and screenshots

### PR DESCRIPTION
**Pull request by microstudi**
This adds the missing documentation and standarizes the linting configuration to make use of Decidim standards

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,10 +71,40 @@ Decidim::Kids.configure do |config|
...
+  config_accessor :minor_authorization_age_attributes do
+    [:birthday, :date_of_birth, :birth_date, :birthdate]
+  end
+  
+  # Other, more advanced, configuration options are available.
+  # Check the source code at lib/decidim/kids.rb for more information.
 end
 ```
 
+## Promoting minor accounts to normal accounts
```
Comment (fblupi) [inline README.md:96]: This section is repeated in the `ARCHITECTURE.md` file mentioned below. We could remove one of the sections

File: .ruby-version [hunk 2]
```diff
@@ -1 +1 @@
-3.0.2
+3.0.5
```
Comment (fblupi) [inline .ruby-version:1]: Why have you changed the ruby version?

**Review by fblupi — COMMENTED** [review]
@microstudi I have left a few comments. Could you check them, please?

File: .ruby-version (hunk 2 above)
Comment (microstudi) [reply .ruby-version:1]: this is the last one, we should use it no?

File: README.md (hunk 1 above)
Comment (microstudi) [reply README.md:96]: ok, remove it.

**Review by fblupi — APPROVED** [review]

**Pull request merged**